wiki/fabrication/ultimaker-series.md

```diff
@@ -0,0 +1,100 @@
+- 2 x Ultimaker 3 (no longer manufactured by Ultimaker)
+- 2 x Ultimaker S3
+
+
+## Software
+The Ultimaker Series of printers uses the [Ultimaker Cura](https://ultimaker.com/software/ultimaker-cura) software to slice STL files for printing. When setting up a print, it is important to crosscheck your print settings with the printer you have on hand. This is expanded on below.
+
+
+## Setting up a print
+Using your favorite CAD Sofware (i.e. Solidworks), prepare an STL file of your model for printing. Open up CURA, and use the folder icon in the top left of your window (Circle 1) to select the STL file(s) you have prepared.
+
+Using the options on the left side of the screen (Circle 2), move your piece(s) around. Using these controls, you can change the orientaiton of the part, as well as its location on the build plate. Make sure that part is oriented to minimize overhangs, and if you have multiple pieces, that the pieces are not too close to each other.
+
+<figcaption align = "center"><b>Figure 1: Ultimaker Cura PREPARE screen when preparing a print. </b></figcaption>
+
+![Ultimaker Cura Screen](assets/ultimaker_fig1.png)
+
+Next, check that the material in the menu indicated by Circle 3 **matches what is actually loaded into your printer**. If you don't do this, you your spliced file may not load correctly when you attempt to start the print.
+
+In the menu indicated by Circle 4, there are a variety of print options for you to explore. You can learn more about different print options and how to choose them for your specific piece by heading over to our page on [Fabrication Considerations for 3D Printing](/wiki/fabrication/fabrication_considerations_for_3D_printing/).
+
+Finally, you can click the 'Slice' button indiciated by Circle 5 when ready. As long as your material was loaded correctly, you should not have to worry about re-setting things like print temperatures or changing the core. You *should* check those when changing the loaded material spools, however.
+
+## Previewing your part before printing
+The preview screen is a last look at your part and what the print will look like both on the outside and on the inside.
+
+<figcaption align = "center"><b>Figure 2: Ultimaker Cura PREVIEW screen. </b></figcaption>
+
+![Ultimaker Cura Screen](assets/ultimaker_fig2.png)
+
+Following the circles in Figure 2, this is what to look out for on your print:
+
+**6** - Use the sliding bar on the right side of the screen to look at a crossection of your part at different heights. This can give you a look inside to the infill of your part, and how the infill looks around internal features like holes (as shown in figure 2). Use this to catch potential failures before printing.
+
+**7** - Take a look at how long and how much material this print will take  *at the current print settings*. If either of these numbers are unsuitable to you, hope back over to the top bar menu and change your print settings till these numbers are to your liking!
```

Comment:
There is a change from a `(Circle x)` format of referencing the markup in the figures to a numbered list format here - as a reader I'm confused why all of a sudden a numbered list appears starting at point 6! Consider revising.